src/Helper/Service/AssetFactory.php

```diff
@@ -2,17 +2,14 @@
 
 namespace Laminas\View\Helper\Service;
 
-use Interop\Container\ContainerInterface;
-use Laminas\ServiceManager\FactoryInterface;
 use Laminas\ServiceManager\ServiceLocatorInterface;
 use Laminas\View\Exception;
 use Laminas\View\Helper\Asset;
+use Psr\Container\ContainerInterface;
 
-class AssetFactory implements FactoryInterface
+class AssetFactory
```

Comment:
We should re-add the factory here as removing existing factories might cause trouble in upstream projects.
Even tho, that I dont think factories should be extended (and thats why I always end up adding `final` to em), I've already seen factories which actually do extend other factories and thus changing anything here could be problematic.

I think removing the interface is fine until we keep the method signature.
Changing to PSR-11 container might be fine when bumping to PHP 7.4